components/support/ktx/src/androidTest/java/mozilla/components/support/ktx/android/view/WindowKtTest.kt

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.ktx.android.view
+
+import android.graphics.Color
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import mozilla.components.support.ktx.MainActivity
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Rule
+import org.junit.Test
+
+class WindowKtTest {
+    @get:Rule
+    val activityRule: ActivityScenarioRule<MainActivity> = ActivityScenarioRule(MainActivity::class.java)
+
+    @Test
+    fun When_A_Light_Color_Is_Applied_To_Status_Bar_Then_Set_Is_Appearance_Light_Status_Bars_To_True() {
```

Comment:
Nit: For other UI tests we are using camel case convention see [SmokeTests](https://github.com/mozilla-mobile/android-components/blob/2d5d1ef117551ebd69c5d52582b59cb59498e9f9/samples/browser/src/androidTest/java/org/mozilla/samples/browser/SmokeTests.kt#L51) :)

Reply:
Done